lenskit-eval/src/main/java/org/lenskit/eval/traintest/DataSetBuilder.java

```diff
@@ -41,6 +45,8 @@
     private Map<String, Object> attributes = new LinkedHashMap<>();
     private StaticDataSource queryData;
     private UUID isoGroup = new UUID(0, 0);
+    private List<EntityType> entityTypes;
+    private List<EntityType> ratingsList = Lists.newArrayList(CommonTypes.RATING);
```

Comment:
That default value should just be assigned to `entityTypes`, not a separate field. We never use `ratingsList`.